docs/user/overview.md

```diff
@@ -60,15 +60,15 @@ mv initial/install-config.yaml .
 rm -rf initial
 ```
 
-You can use the saved install-config for future clusters by copying it that install config into the asset directory and then invoking the installer:
+You can use the saved install-config for future clusters by copying it i nto the asset directory and then invoking the installer:
 
 ```sh
 mkdir cluster-0
 cp install-config.yaml cluster-0/
 openshift-install --dir=cluster-0 create cluster
 ```
 
-Supplying a previously-generated install-config like this is [explicitly part of the stable API](versioning.md).
+Supplying a previously-generated install-config like this is [explicitly part of the stable API](versioning.md). Do note that the copied install-config would be deleted from the asset directory by the installer.
```

Comment:
"Do note" -> "note".

Reply:
I'd prefer to use the term "consume". The information isn't lost, it's just been moved.

Reply:
I think "[at any point before running delete cluster][1]" is an important point to make.  We don't want to give a false sense of security.

[1]: https://github.com/openshift/installer/pull/1122#discussion_r250426690